src/BrandwatchReactAuth.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import jwtDecode from 'jwt-decode';
+import TokenStore from 'viziaauth';
+
+export default class BrandwatchReactAuth extends Component {
+  getChildContext() {
+    return {
+      brandwatchAuthLogout: this.handleLogout,
```

Comment:
Good idea... context now looks like.

```js
  getChildContext() {
    return {
      brandwatchAuthLogout: this.handleLogout,
      brandwatchAuthGetProfile: this.handleGetProfile,
      brandwatchAuthGetToken: this.handleGetToken,
    };
  }
```

... and have nuked `onAuthenticate` 👍